src/components/Header.js

```diff
@@ -1,4 +1,4 @@
-import headerStyles from "../styles/Header.module.css";
```

Comment:
Replace all files under the `components` dir with a placeholder `.gitkeep` file so that `git` at least know it's supposed to version control "empty" directories as well.

Reply:
basically, should i remove all the files under components and pages and add a .gitkeep files to track the empty dir ?

Reply:
Yes, that's the idea. I know I can be nit picky at times 😅 but please bear with me on this. Also just a reminder remove any other unneeded files (if any exists on your fork) as well.

Reply:
no i was also thinking to remove them because they doesn't make sense kept it as it was already there. lol

Reply:
Yup, we plan on starting from level ground so cleaning out as much fluff as we can right now is the way to go.